src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -108,6 +108,21 @@ public class ImportAzureKeyVaultCertificate : KeyVaultCmdletBase
                     HelpMessage = "Specifies the password for the certificate and private key base64 encoded string to import.")]
         public SecureString Password { get; set; }
 
+        /// <summary>
+        /// File Path
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "A file path to specify management policy for the certificate that contains JSON encoded policy definition.")]
```

Comment:
Update the description of PolicyPath and PolicyObject to demonstrate they are mutual-exclusive.